docs/concepts/content-addressing.md

```diff
@@ -7,7 +7,7 @@ description: Learn about how content addressing works and how content identifier
 # Content addressing and CIDs
 
 ::: tip
-If you're interested in how content addressing fits into how IPFS works with files in general, check out this video from IPFS Camp 2019! [Core Course: How IPFS Deals With Files](https://www.youtube.com/watch?v=Z5zNPwMDYGg)
+For a deep dive into content addressing and how it works, take a look at the ProtoSchools course on the [Anatomy of a CID](https://proto.school/#/anatomy-of-a-cid).
```

Comment:
```suggestion
For a deep dive into how Content Identifiers (CIDs) are constructed, take a look at ProtoSchool's tutorial on the [Anatomy of a CID](https://proto.school/#/anatomy-of-a-cid).
```

The tutorial isn't about content addressing in general (Decentralized Data Structures is closer to that), it's just about the CID itself and its gazillion self-referential parts. I'd recommend avoiding "course" because it's not what we call them and we may later create a thing called a course that's a collection of tutorials. ¯\_(ツ)_/¯

Reply:
Cool beans. I'll follow your example here then :)